python/python/glide/__init__.py

```diff
@@ -1,6 +1,23 @@
 # Copyright GLIDE-for-Redis Project Contributors - SPDX Identifier: Apache-2.0
 
-from glide.async_commands.bitmap import BitmapIndexType, BitwiseOperation, OffsetOptions
+from glide.async_commands.bitmap import (
+    BitEncoding,
```

Comment:
do you need to add this to `__all__` section?

Reply:
Good catch, thanks